File: python/PrimeNumber.py
#Function to check whether it is prime number or not.
#        if prime
#            return true;
#        else
#            return false;
def isPrime(n):
    if n<2:
        return False
    for j in range(2,n):
        if n%j==0:
            return False

    return True

#To check whether it is main class or not
if(__name__=="__main__"):
    #Get input from user as integer
    user_input = int(input("Enter the number to find the number of prime numbers:"))
    for i in range(1,user_input):
        if isPrime(i):
            print(i)

File: java/PrimeNumber.java
package primenumber;

import java.util.Scanner;

/**
 *
 * @author Vinoth
 */
public class PrimeNumber {

    /*
    Function to check whether it is prime number or not.
        if prime 
            return true;
        else
            return false;
    */
    public static boolean isPrime(int n) {
        if (n < 2) {
            return false;
        }

        for (int i = 2; i < n; i++) {
            if (n % i == 0) {
                return false;
            }
        }

        return true;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number to find the number of prime numbers: \n");
        int input = sc.nextInt(); //Get input from the user
        for (int j = 1; j <= input; j++) {
            if (isPrime(j)) {
                System.out.println(j);
            }
        }
    }
}
